src/language-graphql/printer-graphql.js

```diff
@@ -572,7 +574,11 @@ function printInterfaces(path, options, print) {
   return parts;
 }
 
-function clean(/*node, newNode , parent*/) {}
+function clean(node, newNode /* , parent */) {
+  if (node.kind === "StringValue" ) {
```

Comment:
```suggestion
  if (node.kind === "StringValue") {
```

Reply:
This will remove the failing linting task.

Reply:
I do not have permission to push to the repo so I can fix the task at this period of time